test/sanity/README.md

```diff
@@ -1,52 +1,45 @@
-# Kubernetes Sanity Script Test
+# Sanity Tests for CSI PowerFlex
 
 This test runs the Kubernetes sanity test at https://github.com/kubernetes-csi/csi-test.
-The version was v3.1.0
 
-To run the test:
-
-	1. run `git clone https://github.com/kubernetes-csi/csi-test.git`
-	2. Check out tag v3.1.0
```

Comment:
Minor comment, is v3.1.0 the latest as of now. In case there is a newer one in future, we could simply say checkout the latest tag (for ex. 3.1.0 as of july 2022). OR should we just always reference 3.1.0 since this is the version we have tested against.

Reply:
You might be referring to the removed changes. In the updated documentation (green), I removed the specification of any `csi-test` version.